tests/scripts/pkgconfig.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+#
+# Purpose
+#
+# Test pkgconfig files.
+#
+# For each of the build pkg-config files, .pc files, check that
+# they validate and do some basic sanity testing on the output,
+# ie. that the strings are non-empty.
+#
+# NOTE: This requires the built pc files to be on the pkg-config
+# search path, this can be controlled with env variable
+# PKG_CONFIG_PATH. See man(1) pkg-config for details.
+#
+
+set -e -u
+
+all_pcs="mbedtls mbedx509 mbedcrypto"
+
+for pc in $all_pcs; do
```

Comment:
Any reason why not to do a `find(1)` even just a glob?

Reply:
A couple of reasons why. One could search for it, but there is a difference between the package config file vs the package name. Ie the pc file is `mbedtls.pc`, you would need to `basename` it. However, the name of that file is crucial, we don't want someone to think they can arbitrarily rename it, as it would break folks using package config. Hardcoding it on the test side ensures a failure on a rename. I think a comment there explaining this so if it does fail, a person can make a sane choice whether or not the rename side effects are intentional.